PluginsAndFeatures/azure-toolkit-for-intellij/src/com/microsoft/intellij/runner/webapp/webappconfig/ui/WebAppSettingPanel.java

```diff
@@ -542,7 +537,8 @@ public void enableDeploymentSlotPanel() {
 
     private void toggleDeployPanel(boolean isUsingExisting) {
         pnlExist.setVisible(isUsingExisting);
-        pnlDeploySlot.setVisible(isUsingExisting);
+        deployToSlotCheckBox.setVisible(isUsingExisting);
```

Comment:
It's related with "new webapp" / "existing webapp". Nothing to do with the slots.

Reply:
Yes, move the pnlSlot to pnlExisting, then we can manage the elements visibility in one entry.